packages/core/src/components/popover/popover.tsx

```diff
@@ -185,6 +185,12 @@ export interface IPopoverProps extends IOverlayableProps, IProps {
      */
     targetClassName?: string;
 
+    /**
+     * Component element of the popover target.
+     * @default "div"
+     */
+    targetComponent?: string;
```

Comment:
rename to `targetElementTag` to match `rootElementTag`, and update docs to match the existing prop.